app/views/users/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<!--<%= form_tag("/users", method: "post") do %>
+  <%= label_tag(:username, "Username:") %>
+  <%= text_field_tag(:username) %><br>
+
+  <%= label_tag(:email, "email:") %>
+  <%= text_field_tag(:email) %><br>
```

Comment:
It is better is you use `email_filed_tag`.

Reply:
Thanks!